pkg/ruler/base/ruler.go

```diff
@@ -587,19 +587,19 @@ func (r *Ruler) listRulesShuffleSharding(ctx context.Context) (map[string]rulesp
 	g, gctx := errgroup.WithContext(ctx)
 	for i := 0; i < concurrency; i++ {
 		g.Go(func() error {
-			for userID := range userCh {
-				groups, err := r.store.ListRuleGroupsForUserAndNamespace(gctx, userID, "")
+			for orgID := range userCh {
+				groups, err := r.store.ListRuleGroupsForUserAndNamespace(gctx, orgID, "")
 				if err != nil {
-					return errors.Wrapf(err, "failed to fetch rule groups for user %s", userID)
+					return errors.Wrapf(err, "failed to fetch rule groups for user %s", orgID)
```

Comment:
Same here. Should it be
```suggestion
					return errors.Wrapf(err, "failed to fetch rule groups for org %s", orgID)
```